"just test" fails

### Describe the bug

`just build` and `just build-bin` work great, however `just test` fails.
I confirmed this for both `master` and `0.2.0` on my Ubuntu machine.

### Steps to reproduce

run `just test` in the top directory


This can be reproduced minimally by:

```
cd wasmer
go test test/imports.go 
```

### Expected behavior

I assume tests pass.

### Actual behavior

output:
```
just test
# command-line-arguments
In file included from _cgo_export.c:4:0:
cgo-gcc-export-header-prolog:46:16: error: conflicting types for ‘sum_i64’
In file included from _cgo_export.c:4:0:
imports.go:6:17: note: previous declaration of ‘sum_i64’ was here
_cgo_export.c:46:9: error: conflicting types for ‘sum_i64’
 GoInt64 sum_i64(void* p0, GoInt64 p1, GoInt64 p2)
         ^~~~~~~
In file included from _cgo_export.c:4:0:
imports.go:6:17: note: previous declaration of ‘sum_i64’ was here
FAIL    command-line-arguments [build failed]
```

### Additional context

I was trying to build it all from source and use a custom *.so. I thought I caused the error, but even after checking out `0.2.0` and running `git clean -xcf` and `rm -rf target` I still get the error.

--- a/wasmer/test/imports.go
+++ b/wasmer/test/imports.go
@@ -3,7 +3,7 @@ package wasmertest
 // #include <stdlib.h>
 //
 // extern int32_t sum(void *context, int32_t x, int32_t y);
-// extern int64_t sum_i64(void *context, int64_t x, int64_t y);
+// extern long long sum_i64(void *context, long long x, long long y);
 // extern float sum_f32(void *context, float x, float y);
 // extern double sum_f64(void *context, double x, double y);
 // extern int32_t missingContext();

